enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -203,80 +205,49 @@ metadata:
-  * But then the same concern about avoiding unnecessary reconciliation remains, as in the [previous section](#configuring-custom-trusted-certificates): 
-    how to detect if the trust bundle file changed or the operator just restarted for another reason?
 ---
 
 ### Test Plan & Infrastructure Needed
 
-In addition to unit testing individual WMCO packages and controllers, an e2e will be added to the WMCO repo for the
-master/release-4.14 branches. We will leverage the [existing proxy workflow](https://steps.ci.openshift.org/workflow/openshift-e2e-aws-proxy)
-from the release repo and add a job to the WMCO repo's CI config to test with. We will run regression testing to make
-sure the addition of the egress proxy feature does not break existing functionality. If we release a community offering
-with this feature, we will need to add a CI job using a cluster-wide proxy on OKD.
+In addition to unit testing individual WMCO packages and controllers, an e2e job will be added to the release repo for
+WMCO's master/release-4.14 branches. This can replace the current vSphere job to avoid growing the number of required jobs.
 
-### Open Questions
+We create a new workflow to test on vSphere. The [indivudial steps](https://github.com/openshift/release/tree/master/ci-operator/step-registry/ipi/conf/vsphere/proxy/https) 
+are already present in the release repo and used by other jobs; we will string them together as needed to satisfy all
+our testing requirements (OVN and HTTPS proxy secured by additional certs). The workflow will have the following steps:
+```yaml
+Pre:
+- ipi-conf-vsphere
+- ipi-conf-vsphere-proxy-https
+- ovn-conf
+- ovn-conf-hybrid-manifest-with-custom-vxlan-port
+- ipi-install-vsphere
+Test:
+- windows-e2e-operator-test-with-custom-vxlan-port
+Post:
+- gather-network
+- gather-proxy
+- ipi-vsphere-post
+```
 
-> 1. Is testing cluster-wide proxy in CI on __only AWS__ okay? This is consistent with what the Linux side does.
-     QE may want to cover all platforms though.
-> 2. Can we use the existing [CI proxy job](https://github.com/openshift/release/tree/master/ci-operator/step-registry/ipi/conf/aws/proxy)
-     to test an HTTPS proxy that requires custom certs to access, or can we only test HTTP and no-proxy using it? If the
-     latter, we'll need to explore how to do this in EC2 and add improvements to the linked workflow.
-> 3. Do user-created Windows workloads need proxy config set in their spec? Does this already happen for Linux
-     counterparts? e.g. setting proxy env vars and mounting the trustedCA ConfigMap as a volume to the deployment spec
-     Regardless, I don't think this should be WMCO's responsibility, so is this on another team or upon the user?
-  <!-- TODO: update non-goals based on the answer to number 3  -->
+We will run regression testing to make sure the addition of the egress proxy feature does not break existing functionality.
```

Comment:
We or QE? I would replace the `We` with `Existing e2e tests will ensure the addition...`